### PR TITLE
Trim unused footer props/vars

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** The Gradebook currently passes a lot of unused info to the footer. Mercilessly purge them from the codebase.

JIRA: [EDUCATOR-5773](https://openedx.atlassian.net/browse/EDUCATOR-5773)

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

Verify that footer behavior/styling is unchanged:
- [ ] Footer shows "Powered by Open edX" logo
- [ ] Footer links to Open edX site

Screen Shot 2021-06-03 at 16 07 29

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -13,48 +13,13 @@ import {
...
-import {
-  faFacebookSquare,
-  faTwitterSquare,
-  faLinkedin,
-  faRedditSquare,
-} from '@fortawesome/free-brands-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-
-import SiteFooter, { messages as footerMessages } from '@edx/frontend-component-footer';
+import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
```
Comment (leangseu-edx) [inline src/index.jsx:16]: `footerMessages` is an unused variable?

File: src/index.jsx [hunk 2]
```diff
@@ -98,8 +43,6 @@ subscribe(APP_READY, () => {
 });
 
 initialize({
-  messages: [
-    footerMessages,
-  ],
+  messages: [],
```
Comment (jansenk) [inline src/index.jsx:46]: What is this? Is it still necessary?

**Review by jansenk — APPROVED** [review]
rip and tear until it is done

**Review by leangseu-edx — APPROVED** [review]

File: src/index.jsx (hunk 2 above)
Comment (nsprenkle) [reply src/index.jsx]: `initialize` fails without `messages` arg. But looking into it, turns out we actually should keep them in. Reverting.

File: src/index.jsx (hunk 1 above)
Comment (nsprenkle) [reply src/index.jsx:16]: ^ It is used further down in the `initialize` script

**Review by muselesscreator — APPROVED** [review]
LGTM.  :ship: :it:
